docs/080-breaking-changes.rst

```diff
@@ -155,3 +156,15 @@ Interface Changes
   Use the "compact AST" (``--ast-compact--json`` resp. ``AST``) as replacement.
 
 * The old error reporter (``--old-reporter``) has been removed.
+
+
+How to update your code
+=======================
```

Comment:
Don't we also want to mention some of the less likely problems, like removing `log` functions or adding parentheses over exponentiation if anyone depended on the old operator order?

Especially the latter will just silently start producing wrong results instead of failing.